server/src/main/java/org/elasticsearch/index/IndexService.java

```diff
@@ -19,9 +19,30 @@
 
 package org.elasticsearch.index;
 
-import io.crate.common.annotations.VisibleForTesting;
-import io.crate.common.io.IOUtils;
-import io.crate.common.unit.TimeValue;
+import static io.crate.common.collections.MapBuilder.newMapBuilder;
```

Comment:
Is this correct / new style? to have static imports at the top?

Reply:
I think it should be, if it isn't in intellij we have to adjust the `codeStyle.xml` further to have intellij/eclipse match.